diff --git a/node_modules/vite/dist/node/chunks/dep-5e7f419b.js b/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
index ff6c927..ab76226 100644
--- a/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
+++ b/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
@@ -35227,8 +35227,8 @@ function addToHTMLProxyTransformResult(hash, code) {
 const assetAttrsConfig = {
     link: ['href'],
     video: ['src', 'poster'],
-    source: ['src', 'srcset'],
-    img: ['src', 'srcset'],
+    source: ['src', 'srcset', 'data-src', 'data-srcset'],
+    img: ['src', 'srcset', 'data-src', 'data-srcset'],
     image: ['xlink:href', 'href'],
     use: ['xlink:href', 'href'],
 };
@@ -35518,7 +35518,7 @@ function buildHtmlPlugin(config) {
                         !namedOutput.includes(content.replace(/^\//, '')) // Allow for absolute references as named output can't be an absolute path
                     ) {
                         try {
-                            const url = attr.prefix === undefined && attr.name === 'srcset'
+                            const url = attr.prefix === undefined && (attr.name === 'srcset' || attr.name === 'data-srcset')
                                 ? await processSrcSet(content, ({ url }) => urlToBuiltUrl(url, id, config, this))
                                 : await urlToBuiltUrl(content, id, config, this);
                             overwriteAttrValue(s, sourceCodeLocation, url);
@@ -60566,7 +60566,7 @@ const processNodeUrl = (attr, sourceCodeLocation, s, config, htmlPath, originalU
         // path will add `/a/` prefix, it will caused 404.
         // rewrite before `./index.js` -> `localhost:5173/a/index.js`.
         // rewrite after `../index.js` -> `localhost:5173/index.js`.
-        const processedUrl = attr.name === 'srcset' && attr.prefix === undefined
+        const processedUrl = (attr.name === 'srcset' || attr.name === 'data-srcset') && attr.prefix === undefined
             ? processSrcSetSync(url, ({ url }) => replacer(url))
             : replacer(url);
         overwriteAttrValue(s, sourceCodeLocation, processedUrl);
